test/Connector.t.sol

```diff
@@ -290,6 +290,12 @@ contract ConnectorTest is Test {
         token.approve(address(bridgedConnector), amount);
         bridgedConnector.transfer(poolId, trancheId, ConnectorMessages.Domain.Centrifuge, centChainAddress, amount);
         assertEq(token.balanceOf(address(this)), 0);
+
+        // Finally, verify the connector called `router.send`
+        bytes memory message = ConnectorMessages.formatTransfer(
+            poolId, trancheId, ConnectorMessages.formatDomain(ConnectorMessages.Domain.Centrifuge), centChainAddress, amount
+        );
+        assert(mockXcmRouter.sentMessages(message));
```

Comment:
This is using the Solidity `assert` rather than Foundry's testing assert. Should probably be `assertEq(mockXcmRouter.sentMessages(message), true);`

(seems like there's no `assertTrue`: https://book.getfoundry.sh/reference/forge-std/std-assertions)

Reply:
Done 👍